.circleci/config.yml

```diff
@@ -51,88 +51,3 @@ workflows:
           filters:
             tags:
               only: /^v.*/
-
-      - architect/push-to-app-collection:
-          context: architect
-          name: push-node-exporter-to-shared-app-collection
-          app_name: "node-exporter-app"
-          app_namespace: "monitoring"
-          app_collection_repo: "shared-app-collection"
-          requires:
-            - push-node-exporter-to-control-plane-app-catalog
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /^v.*/
-
-      - architect/push-to-app-collection:
-          context: architect
-          name: push-node-exporter-to-aws-app-collection
-          app_name: "node-exporter-app"
-          app_namespace: "monitoring"
-          app_collection_repo: "aws-app-collection"
-          requires:
-            - push-node-exporter-to-control-plane-app-catalog
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /^v.*/
-
-      - architect/push-to-app-collection:
```

Comment:
Only CAPI collections should be removed

Reply:
But, default apps is already deploying node-exporter for other providers which will conflict it's install again with app collection. Isn't that so?
- https://github.com/giantswarm/default-apps-vsphere/blob/main/helm/default-apps-vsphere/values.yaml
- https://github.com/giantswarm/default-apps-cloud-director/blob/main/helm/default-apps-cloud-director/values.yaml
- https://github.com/giantswarm/default-apps-aws/blob/master/helm/default-apps-aws/values.yaml

Exception is for azure where it's not part of default apps.
- https://github.com/giantswarm/default-apps-azure/blob/main/helm/default-apps-azure/values.yaml

Reply:
default apps aws is for capa, not aws (i.e. vintage)

Reply:
Ah okay, got it. Thanks much! ❤️